system/views/formbuilder/layouts/core/formLayout.cfm

```diff
@@ -20,18 +20,57 @@
 					var $form = $('###args.id#');
 					<cfif Len( Trim( args.validationJs ) )>
 						if ( typeof jQuery.validator !== 'undefined' ) {
+
 							$form.validate( #args.validationJs# );
 							$form.on('submit', function () {
+
```

Comment:
This validation logic should go through our validation framework and not put just in this specific page. Ping me and I'll show you where and how this can be done.